### DISCUSSION TITLE
Help with adding a for loop in my schema markup

### DISCUSSION THREAD

**Discussion by hope-elijah** [BioSchemas/specifications#600] [Q&A]
Hello everyone. I need a little bit of guidance using the bioschema course profile. I am trying to do a for loop in my hasCourseInstance but I keep getting an error of unparsable structured data whenever I try to validate my markup.

The snippet looks like this:

`
"hasCourseInstance" : [
{%- for w in schedule.weeks %}
{
  "@context": "https://schema.org",
      "@type": "CourseInstance",
      "dct:conformsTo": "https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06",
      "courseMode": "['online', 'synchronous']",
      "startDate" : "{{c.date}}",
      "endDate" :"{{c.date}}",
      "duration": "{{c.date}}",
      "name" : "{{c.title}}"
}
]
`

**Comment by AlasdairGray** [comment]
What is the rendered markup that comes out of your for loop?

**Reply by ljgarcia** [reply]
Hi @DOC-ELIJAH. Have you sorted it out? In general terms, it is difficult to find out issues with implementations as different languages and approaches can be used there, same as different data models. It looks like you are using Jekyll but some JavaScript frameworks look similar. I would suggest using the mailing list next time as more people follow that one.
